src/clj/game/core/access.clj

```diff
@@ -748,7 +753,9 @@
                     (and pos-max?
                          pos-total-cards?
                          only-card)
-                    (access-card state side eid (first total-cards))
+                    (if (= (second (get-zone only-card)) :rd)
```

Comment:
This disallows choosing the card in root, which you can do if you're accessing a central. The list of candidates for centrals includes all cards in root and a set number of cards from the central itself. If a card is in R&D itself, the zone will be `[:deck]`.

Reply:
Hmm. There aren't any effects that set the only card to access to be something that's not on the field, but it's probably still worth cleaning this up.

Reply:
I think changing it to be something like `(if (or (in-deck? only-card) (in-rd-root? only-card)) ...)` would be fine, and then the same for the other checks.

Reply:
(I also prefer that we use the card.cljc predicate functions instead of looking at the contents of `:zone`, cuz it's semantic and allows us to change it if we ever decide to.

Reply:
Sick, did that. I added a helper in card.cljc for `in-remote-root?` too, to be consistent.